### PR TITLE
Add Sentry

### LINKED ISSUE #76: Sentry for APM and Issues

### PR DESCRIPTION
**Pull request by nvernooy**
Add the sentry packages to the requirements file
Add SENTRY_DSN to the kubernetes configmap
Update the README
Closes #76

### CODE REVIEW

File: {{cookiecutter.project_slug}}/README.md [hunk 1]
```diff
@@ -72,3 +72,18 @@ To create a superuser use the following commands:
...
+
+Sentry can be used for error reporting at the application level. The packages and SENTRY_DSN configuration 
+are available in the project, but the project will still need to be manually added to Sentry.
+
+Basic steps:
+
+1. Create the project in [Sentry](https://sixfeetup.sentry.io/projects/)
+2. Configure Slack notifications
+3. Add team members in Sentry
+4. Add code to project
```
Comment (rochecompaan) [inline {{cookiecutter.project_slug}}/README.md:86]: How do you do this? What code?

File: {{cookiecutter.project_slug}}/README.md [hunk 2]
```diff
@@ -72,3 +72,18 @@ To create a superuser use the following commands:
...
+5. Update `django.configmap.yaml` SENTRY_DSN with the DSN provided in the Sentry project for this environment
+
+For more detailed steps view the [SFU Paper page](https://paper.dropbox.com/doc/Adding-Sentry-to-Projects--B_BGoXS6w~Q8FbkaVOlRooaXAg-7sXkQxTexTtV6qT367VGJ)
```
Comment (rochecompaan) [inline {{cookiecutter.project_slug}}/README.md:89]: You can't include links to private company resources. Cookie-cutter is an open-source project. I recommend we generalize the paper page and add it as documentation to the cookie-cutter.

File: {{cookiecutter.project_slug}}/k8s/base/django.configmap.yaml [hunk 3]
```diff
@@ -27,3 +27,6 @@ data:
   AWS_S3_ACCESS_KEY_ID: CHANGEME_S3_IAM_USER
   DJANGO_AWS_S3_REGION_NAME: {{cookiecutter.aws_region}}
   DJANGO_AWS_STORAGE_BUCKET_NAME: "CHANGEME_S3_BUCKET_NAME"
+
+  # sentry monitoring
+  SENTRY_DSN: "CHANGEME_SENTRY_KEY"
```
Comment (rochecompaan) [inline {{cookiecutter.project_slug}}/k8s/base/django.configmap.yaml:32]: I really don't like CHANGEME values ... They often don't get changed.
```suggestion
  SENTRY_DSN: ""
```

**Review by rochecompaan — CHANGES_REQUESTED** [review]

**Review by rochecompaan — APPROVED** [review]
Nice!

**Comment by balavec** [comment]
@rochecompaan do we also want to add sentry to the react frontend?

**Comment by rochecompaan** [comment]
@balavec yes you're right, we do want to add it to React. @nvernooy do you mind creating an issue to add SENTRY to react since you have already merged this PR?

**Pull request merged**